apps/finance/contracts/Finance.sol

```diff
@@ -108,15 +108,15 @@ contract Finance is AragonApp {
             this.balance,
             "Ether transfer to Finance app"
         );
-        vault.deposit.value(this.balance)(ETH, msg.sender, this.balance, new bytes(0));
+        vault.deposit.value(this.balance)(ETH, this, this.balance);
```

Comment:
Just to give some visibility: I think it's safe to pass along `msg.sender` here if we remove the check in the vault (see below)

Reply:
What would be the point of passing `msg.sender`?

Reply:
Then we need senders to create the allowance to the Vault, which would allow to bypass the Finance deposit by calling deposit directly in the vault.

Reply:
@bingen passing `msg.sender` preserves the actual sender of the funds (rather than finance) when the vault emits its `Deposit` event.

@izqui this would only be for ETH, so no allowance required.

Reply:
Yes, it's just that I don't see the point of it, as this is kind of a escape hatch mechanism, so I think it makes sense to see funds coming from Finance. Finance is already recording incoming transaction with the actual sender, so it can be tracked there. But no strong opinion about it, I'm fine if you want to change it.